content/docs/buildpack-author-guide/create-buildpack/building-blocks-cnb.md

```diff
@@ -27,57 +43,47 @@ api = "0.5"
 
 # Buildpack ID and metadata
 [buildpack]
-id = "examples/ruby"
-version = "0.0.1"
-name = "Ruby Buildpack"
+  id = "examples/ruby"
+  version = "0.0.1"
 
 # Stacks that the buildpack will work with
 [[stacks]]
-id = "io.buildpacks.samples.stacks.bionic"
+  id = "io.buildpacks.stacks.bionic"
```

Comment:
I believe we explicitly wanted the samples stack id for the tutorial so that users don't end up accidentally using these buildpacks in production.

Reply:
Yeah Sure !